src/routes/index.ts

```diff
@@ -0,0 +1,8 @@
+import express from '@feathersjs/express';
+
+const router = express.Router();
+
+// Add routes here
+router.use('/', express.static('public'));
```

Comment:
shouldn't we have more routes here? I'm also trying to figure out how Feather.js works but perhaps routes and their handlers are called services, so maybe we could move that here?

What else do you think we can move to this file @mstanciu552 ?

Reply:
We should have more routes.
The problem is that I didn't know what routes to make as I don't fully understand what the API should do.

This is just a structure for adding more routes.

Reply:
I see, then @eddiejaoude could you confirm if services are the same as routes in Express? Just trying to figure out if we'll have more routes in the future, or if we just add a new service.

Reply:
I had to remind myself from the featherjs docs, I think we will register services not routes. Services each will manage their own routes. So maybe the idea is good but the naming needs changing?

For example...

**Service**
```ts
import { Application, Id, NullableId, Params, Service } from '@feathersjs/feathers';

class MyService implements Service<any> {
  async find(params: Params) {}
  async get(id: Id, params: Params) {}
  async create(data: any, params: Params) {}
  async update(id: NullableId, data: any, params: Params) {}
  async patch(id: NullableId, data: any, params: Params) {}
  async remove(id: NullableId, params: Params) {}
}

app.use('/my-service', new MyService());
```

**Register service**
```ts
const app = feathers();

// Register the message service on the Feathers application
app.use('messages', new MessageService());
```

https://docs.feathersjs.com/guides/basics/services.html#service-events

Reply:
Oh so this file should not have multiple routes but a single service that has multiple routes.

Alright I will change that. :+1: